src-tauri/src/main.rs

```diff
@@ -77,14 +81,15 @@ async fn control_grabbing(
 /// Note: this doesn't work on macOS since we cannot give the thread the proper permissions
 /// (will crash on key grab/listen)
 async fn main() {
-    env_logger::init();
+    let log_level: log::LevelFilter = option_env!("RUST_LOG")
+        .and_then(|s| log::LevelFilter::from_str(s).ok())
+        .unwrap_or(log::LevelFilter::Info);
 
-    #[cfg(not(debug_assertions))]
     wooting_macro_backend::MacroBackend::generate_directories();
 
     let backend = MacroBackend::default();
 
-    info!("Running the macro backend");
+    println!("Running the macro backend");
```

Comment:
why `println`?

Reply:
Fixed in 53b377e. I think i assumed that i can't use the logging until it's properly initialized and then forgot to change it back.